test/addons/rook-operator/rook-config-override.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# yamllint disable rule:line-length
+# Based on https://github.com/red-hat-storage/ocs-operator/commit/e39bb415485df8efcfbf8ddee08f545972e7b064
+
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-config-override
+  namespace: rook-ceph
+data:
+  config: |
```

Comment:
These settings can consume log space quickly, so for setups that are retained for prolonged duration I expect these are not ideal. Is there a reason to use these in default envs? Should these be for debug envs optionally?

Reply:
Ilya requested  the full log, see https://tracker.ceph.com/issues/65487?issue_count=296&issue_position=1&next_issue_id=65481

Reply:
I understand that full logs are needed for debugging purposes, hence my question on defaults vs enabling debugging for a particular issue optionally (or in e2e runs, i.e short lived clusters) etc.

Reply:
I can add a flag so the user can opt in only this is needed, maybe --debug option for dernv, passed to the addons using DRENV_DEBUG env variable. What do you think?

Reply:
If we do not have a debug option at present I suggest we proceed with this change and revisit as needed in the future.